app/src/main/java/com/example/concordia_campus_guide/helper/CurrentLocation.java

```diff
@@ -45,9 +45,9 @@ public void run() {
                         setMyLocation(location);
                     }
                 });
-                handler.postDelayed(this, 2000);
+                handler.postDelayed(this, 500);
```

Comment:
Has anyone checked to see if this has performance issues?

Reply:
When I tired it, there wasn't any issues. I asked Viveka and Ferdousara to try it too

Reply:
I tested out the app on an emulator and I am not having any performance issues.